src/character.cpp

```diff
@@ -7281,3 +7281,49 @@ void Character::use_fire( const int quantity )
         return;
     }
 }
+
+int Character::return_heartrate() const
+{
+    //This function returns heartrate in BPM basing of health, physical state, tiredness, moral effects, stimulators and anything that should fit here. 
+    //Some values are picked ti make sense from math point and seem correct but effects may vary in real life
+    //This needs more attention from experienced contributors to work more smooth
+    int average_heartbeat = 70 + rng( -5, 5 ) + rng( -5, 5 );//average healthy is 60-80. That's a simple imitation of mormal distribution. There's probably a better way to do that. Possibly this value should be generated with player creation.
```

Comment:
Please put long comments on the line before (preferred) or after a statement, not on the same line

Reply:
Note taken. Most comments will be gone after things are more clear.